data/json/monsters/zed_prisoner.json

```diff
@@ -23,8 +23,8 @@
   {
     "id": "mon_zombie_prisoner_fat",
     "type": "MONSTER",
-    "name": { "str": "fat prisoner zombie" },
-    "description": "This fat zombie wears black and white striped prisoner clothes.",
+    "name": { "str": "obese prisoner zombie" },
+    "description": "This obese zombie wears black and white striped prisoner clothes.",
```

Comment:
```suggestion
    "description": "This zombie's flesh bulges out around it's black and white striped prisoner clothes.  Strange nodules, pustules and growths decorate what skin you can see.",
```

Reply:
> around __its__ black and white

I'm not sure how to suggest on a suggestion.